maintenance/bug_triage.md

```diff
@@ -0,0 +1,49 @@
+# OpenTelemetry Bug Triage
+
+This procedure describes the steps project maintainers and approvers should take to triage a bug report.
+
+Bugs should be created using the [Bug Report](https://github.com/open-telemetry/opentelemetry-js/issues/new?template=bug_report.yaml) issue template.
+This template automatically applies the `bug` and `triage` labels.
+
+### Gather Required Information
+
+The first step is to ensure the bug report is unique and complete.
+If the bug report is not unique, leave a comment with a link to the existing bug and close the issue.
+If the bug report is not complete, leave a comment requesting additional details and apply the `information-requested` label.
+When the user provides additional details, remove the `information-requested` label and repeat this step.
+
+### Categorize
+
+Once a bug report is complete, we can determine if it is truly a bug or not.
+A bug is defined as code that does not work the way it was intended to work when written.
+A change required by specification may not be a bug if the code is working as intended.
+If a bug report is determined not to be a bug, remove the `bug` label and apply the appropriate labels as follows:
+
+- `spec-feature` change required by the specification which adds a new feature or extends an existing feature with new functionality
```

Comment:
These were the obvious labels I could think of, but there may be more we want to add

Reply:
Yeah, I was thinking of adding labels like `traces` and `metrics` (and `logs` in the future) to issues to categorize different signal SDKs. Would these be helpful?